src/main/java/competition/subsystems/turret/commands/TurretWaitForRotationToGoalCommand.java

```diff
@@ -0,0 +1,49 @@
+package competition.subsystems.turret.commands;
+
+import com.google.inject.Inject;
+import competition.subsystems.turret.TurretSubsystem;
+import xbot.common.command.BaseCommand;
+import xbot.common.controls.sensors.XTimer;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.PropertyFactory;
+
+public class TurretWaitForRotationToGoalCommand extends BaseCommand {
+
+    private final TurretSubsystem turretSubsystem;
+    private final DoubleProperty timeoutProperty;
+    private final DoubleProperty errorProperty;
+
+    private double startTime = -1;
+
+    @Inject
+    public TurretWaitForRotationToGoalCommand(PropertyFactory pf, TurretSubsystem tSubsystem) {
+        this.turretSubsystem = tSubsystem;
+        this.addRequirements(this.turretSubsystem);
+
+        pf.setPrefix(this);
+
+        this.timeoutProperty = pf.createPersistentProperty("Timeout", 1);
+        this.errorProperty = pf.createPersistentProperty("Comparision threshold", 1.0);
+    }
+
+    @Override
+    public void initialize() {
+        log.info("Initializing");
+        this.startTime = XTimer.getFPGATimestamp();
+    }
+
+    @Override
+    public void execute() {
+        // This command doesn't do anything
+    }
+
+    @Override
+    public boolean isFinished() {
+        double currentAngle = this.turretSubsystem.getCurrentAngle();
```

Comment:
Seeing this makes me think the maintainer should probably publish the "is subsystem near its goal" directly into the subsystem itself - it's better positioned to check things like:
- Is the system near its goal (what you have here)
- Is the system also at ~0 velocity?
- has the system been near it's goal for some duration (in case we are just zooming past our goal but briefly lined up for one instant)

For now everything here is fine; I'll put an item in the backlog to take a look at that model.